_site/index.html

```diff
@@ -92,30 +92,26 @@ <h3>
             Who?
           </h3>
           <p>
-            The specification is standardized by the TC39 (Technical committee 39) of ECMA (European
-            Computer Manufacturers Association). You can read our <a href="https://ecma-international.org/memento/TC39.htm#Scope">the scope of our work here</a>
+            TC39 is a group of JavaScript developers, implementers, academics, and more, collaborating with the community to maintain and evolve the definition of JavaScript. We are part of <a href="http://www.ecma-international.org/">Ecma International</a>.
           </p>
         </span>
         <span class="basic-info">
           <h3>
             How?
           </h3>
           <p>
-          We have a <a href="https://tc39.github.io/process-document/">four stage process</a> with a
-          rolling release schedule with a new specification published every year.</p> <p>Proposals move
-          through stages at TC39 meetings, which are held every <a href="https://github.com/tc39/agendas/">two months</a>, and the <a href="https://github.com/tc39/tc39-notes/">Notes</a> are made public.
+            We develop the JavaScript (formally, ECMAScript) specification <a href="https://github.com/tc39/ecma262/">on GitHub</a> and at meetings every two months. We follow a <a href="https://tc39.github.io/process-document/">four stage process</a> for new <a href="https://github.com/tc39/proposals/">language feature proposals</a>. See our <a href="https://github.com/tc39/agendas/">meeting agendas</a> and <a href="https://github.com/tc39/tc39-notes/">minutes</a> to learn more.
         </p></span>
         <span class="basic-info">
           
           <h3>
             Want to get involved?
           </h3>
           <p>
-  There are many opportunities to get involved on an individual level. Check out our
-  <a href="(https://github.com/tc39/proposals/blob/master/CONTRIBUTING.md">Contributor Documentation</a> and our guides to learn more.
-
-  To get involved with the committee on an organizational level, read our
-  <a href="https://ecma-international.org/memento/TC39-M.htm">Joining the TC39</a> documentation.
+            TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation, testing, implementations, or even language feature ideas. <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+            See our <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">contributor guide</a> for details. 
+          </p><p>
+            To participate as a TC39 delegate in meetings, <a href="https://ecma-international.org/memento/join.htm">join Ecma</a> and sign our <a href="https://ecma-international.org/memento/Policies/Registration%20Form%20for%20the%20Ecma%20Royalty%20Free%20(RF)%20Task%20Group%20(TG).pdf">patent agreement</a>.
```

Comment:
in a future iteration, we should have delegate as a definable word!

Reply:
Should we reword this as, "To participate in physical TC39 meetings" ?

Reply:
This includes when you’re not physically present for a regular meeting and dial in tho

Reply:
OK, let's just say, "To participate in TC39 meetings", then.